agilent_infiniium_scope.py

```diff
@@ -409,67 +438,69 @@ def _parsePreamble(preamble):
                     ('maxBW',False),
                     ('minBW',False)]
     '''
-    preambleKeys = [('byteFormat',True),
-                ('dataType',True),
-                ('numPoints',True),
-                ('count',True),
-                ('xStep',True),
-                ('xFirst',True),
-                ('xRef',True),
-                ('yStep',True),
-                ('yOrigin',True),
-                ('yRef',True),
-                ('coupling',True),
-                ('xRange',True),
-                ('xLeftDisplay',True),
-                ('yRange',True),
-                ('yCenterDisplay',True),
-                ('date',True),
-                ('time',True),
-                ('model',True),
-                ('acquisitionMode',True),
-                ('percentTimeBucketsComplete',True),
-                ('xUnits',True),
-                ('yUnits',True),
-                ('maxBW',True),
-                ('minBW',True)]
-    preambleDict = {}
-    for key,val in zip(preambleKeys,preambleVals):
+    preamble_keys = [('byteFormat', True),
+                ('dataType', True),
+                ('numPoints', True),
+                ('count', True),
+                ('xStep', True),
+                ('xFirst', True),
+                ('xRef', True),
+                ('yStep', True),
+                ('yOrigin', True),
+                ('yRef', True),
+                ('coupling', True),
+                ('xRange', True),
+                ('xLeftDisplay', True),
+                ('yRange', True),
+                ('yCenterDisplay', True),
+                ('date', True),
+                ('time', True),
+                ('model', True),
+                ('acquisitionMode', True),
+                ('percentTimeBucketsComplete', True),
+                ('xUnits', True),
+                ('yUnits', True),
+                ('maxBW', True),
+                ('minBW', True)]
+    preample_dict = {}
+    for key, val in zip(preamble_keys, preamble_vals):
         if key[1]:
-            preambleDict[key[0]] = val
-    def unitType(num):
-        if num=='1':
+            preample_dict[key[0]] = val
+
+    def unit_type(num):
+        if num == '1':
             return 'V'
-        elif num=='2':
+        elif num == '2':
             return 'ns'
         else:
             raise Exception('Units not time or voltage')
-    preambleDict['xUnit'] = unitType(preambleDict['xUnits'])
-    preambleDict['yUnit'] = unitType(preambleDict['yUnits'])
-    return (preambleDict)
+    preample_dict['xUnit'] = unit_type(preample_dict['xUnits'])
+    preample_dict['yUnit'] = unit_type(preample_dict['yUnits'])
+    return (preample_dict)
+
 
-def _parseBinaryData(data, wordLength):
+def _parseBinaryData(data, word_length):
     """Parse binary data packed as string of RIBinary
     """
```

Comment:
Would be good to include a link to the discussion in the manual of the data format: http://www.keysight.com/upload/cmc_upload/All/9000_series_prog_ref.pdf?&cc=US&lc=eng#page=1091